Task10_Auth/src/test/java/ZipCodesTests.java

```diff
@@ -0,0 +1,70 @@
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ZipCodesTests {
+    @Test
+    public void getZipCodesTest() {
+        ZipCodesClient zipCodesClient = new ZipCodesClient();
```

Comment:
It makes sense to create field of ZipCodesClient type to avoid duplication in tests

Reply:
fixed